turbinia/evidence.py

```diff
@@ -783,7 +794,7 @@ def _preprocess(self, _, required_states):
     # Need to mount parent disk
     if not self.parent_evidence.partition_paths:
       self.parent_evidence.mount_path = mount_local.PreprocessMountPartition(
-          self.parent_evidence.device_path)
+          self.parent_evidence.device_path, self.path_spec.type_indicator)
```

Comment:
Did this come from a merge, or is this a bugfix?

Reply:
Yep, this is a bugfix, the call to PreprocessMountPartition() was missing the last argument